src/instructions/proctored_exam/ProctoredExamInstructions.test.jsx

```diff
@@ -0,0 +1,296 @@
+
+describe('SequenceExamWrapper', () => {
+  it('Start exam instructions can be successfully rendered', () => {
+    store.getState = () => ({
+      examState: {
+        isLoading: false,
+        activeAttempt: null,
+        proctoringSettings: {
+          platform_name: 'Your Platform',
+        },
+        verification: {
+          status: 'none',
+          can_verify: true,
+        },
+        exam: {
+          allow_proctoring_opt_out: true,
+          is_proctored: true,
+          time_limit_mins: 30,
+          attempt: {},
+        },
+      },
+    });
+
+    const { getByTestId } = render(
+      <ExamStateProvider>
+        <Instructions>
+          <div>Sequence</div>
+        </Instructions>
+      </ExamStateProvider>,
+      { store },
+    );
+    expect(getByTestId('start-exam-button')).toHaveTextContent('Continue to my proctored exam.');
+    fireEvent.click(getByTestId('start-exam-without-proctoring-button'));
+    expect(getByTestId('proctored-exam-instructions-title'))
+      .toHaveTextContent('Are you sure you want to take this exam without proctoring?');
+    fireEvent.click(getByTestId('skip-cancel-exam-button'));
+    expect(getByTestId('start-exam-without-proctoring-button'))
+      .toHaveTextContent('Take this exam without proctoring.');
+  });
+
+  it('Instructions are not shown when exam is started', () => {
+    store.getState = () => ({
+      examState: {
+        isLoading: false,
+        verification: {
+          status: 'none',
+          can_verify: true,
+        },
+        activeAttempt: {
+          attempt_status: 'started',
+        },
+        exam: {
+          is_proctored: true,
+          time_limit_mins: 30,
+          attempt: {
+            attempt_status: 'started',
+          },
+        },
+      },
+    });
+
+    const { getByTestId } = render(
+      <ExamStateProvider>
+        <Instructions>
+          <div data-testid="sequence-content">Sequence</div>
+        </Instructions>
+      </ExamStateProvider>,
+      { store },
+    );
+    expect(getByTestId('sequence-content')).toHaveTextContent('Sequence');
+  });
+
+  it('Instructions are shown when attempt status is created', () => {
+    store.getState = () => ({
+      examState: {
+        isLoading: false,
+        proctoringSettings: {
+          platform_name: 'Your Platform',
+        },
+        verification: {
+          status: 'none',
+          can_verify: true,
+        },
+        activeAttempt: {
+          attempt_status: 'created',
+        },
+        exam: {
+          is_proctored: true,
+          time_limit_mins: 30,
+          attempt: {
+            attempt_status: 'created',
+          },
+        },
+      },
+    });
+
+    const { getByTestId } = render(
+      <ExamStateProvider>
+        <Instructions>
+          <div>Sequence</div>
+        </Instructions>
+      </ExamStateProvider>,
+      { store },
+    );
+    expect(getByTestId('exam.VerificationProctoredExamInstructions-continue-button'))
```

Comment:
what about the various verification states? pending/reverify/expired are those covered somewhere?

Reply:
addressed in 152c763